Cargo.toml

```diff
@@ -1,49 +1,47 @@
 [workspace]
 members = [
-    "packages/*",
-    "contracts/*",
-    "tests/polytone-tester"
+  "packages/*",
+  "contracts/*",
```

Comment:
what would happen if we included `accessories/*` here? really my only concern with this PR is that the code layout doesn't match this. if including accessories breaks things, could we perhaps just put a little comment here about how accessories is actually part of the workspace, but isn't included here because of the copying?